src/io/fabric8/Fabric8Commands.groovy

```diff
@@ -496,8 +496,12 @@ def getServiceURL(String serviceName, String namespace = null, String protocol =
 }
 
 def isOpenShiftS2I() {
-    def openshiftYaml = "target/classes/META-INF/fabric8/openshift.yml"
     try {
+        new File('.').eachFileRecurse(FILES) {
+          if(it.name.endsWith('openshift.yml')) {
+          def openshiftYaml = it.absolutePath
```

Comment:
The `def openshiftYaml` here means the rest of the function cant see it.  You can define `openshiftYaml` variable at the top of the method.